salt/modules/pillar.py

```diff
@@ -157,7 +156,7 @@ def get(key,
                     'skipped.', default, ret, type(ret).__name__
                 )
         elif isinstance(default, list):
-            ret = salt.utils.data.traverse_dict_and_list(
+            ret = salt.utils.traverse_dict_and_list(  # pylint: disable=redefined-variable-type
```

Comment:
This actually should be `salt.utils.data.traverse_dict_and_list()`, as salt.utils was split up for the 2018.3 release cycle. If left as is, there will be a deprecation warning.